src/styles.css

```diff
@@ -0,0 +1,1195 @@
+  transition: 0.3s;
+  line-height: 44px;
+}
+
+.overlay a:hover,
+.overlay a:focus {
+  color: #f1f1f1;
+}
+
+.overlay .closebtn {
+  position: absolute;
+  top: 11px;
+  right: 133px;
+  font-size: 30px;
+  font-weight: normal;
+  border: 2px;
+  cursor: pointer;
+}
+
+.content {
+  display: flex;
+  flex-direction: column;
+  background-image: url("../assets/header.png");
+  background-size: 100%;
+  background-repeat: no-repeat;
+  background-color: #fff;
+  width: 100%;
+  height: 90vh;
+  border-radius: 0 0 0 100px;
+  border-bottom: 2px solid #cfd8dc;
+}
+
+.intro {
+  font-family: 'Poppins', sans-serif;
+  font-style: normal;
+  font-weight: 700;
+  font-size: 40px;
+  line-height: 52px;
+  display: flex;
+  align-items: center;
+  color: #172b4d;
+  width: 70%;
+  margin: 5%;
+  margin-top: 25%;
+  animation-name: introin;
+  animation-duration: 4s;
+}
+
+@keyframes introin {
+  from {
+    margin-left: 100%;
+  }
+
+  to {
+    margin-left: 22%;
+  }
+}
+
+textarea::placeholder {
+  color: #b3bac5;
+  padding-top: 15px;
+  font-size: 15px;
+}
+
+input[type="name"]::placeholder {
+  color: #172b4d;
+  font-size: 17px;
+}
+
+input[type="email"]::placeholder {
+  color: #172b4d;
+  font-size: 17px;
+}
+
+.intro1 {
+  font-family: 'Poppins', sans-serif;
+  font-style: normal;
+  font-weight: 400;
+  font-size: 16px;
+  line-height: 24px;
+  position: static;
+  width: 90%;
+  height: 122px;
+  left: 24px;
+  top: 276px;
+  color: #344563;
+  margin-top: 0;
+  margin-top: -40px;
+}
+
+p.intro1 {
+  padding: 15px;
+}
+
+.intro2 {
+  color: #7f8cff;
+  font-family: 'Poppins', sans-serif;
+  font-style: normal;
+  font-weight: bold;
+  font-size: 16px;
+  line-height: 24px;
+  display: flex;
+  align-items: flex-start;
+  padding-left: 15px;
+  padding-top: 35px;
+  grid-gap: 30px;
+  justify-items: center;
+  margin-left: 20%;
+}
+
+.social {
+  display: flex;
+  list-style: none;
+}
+
+ul.social {
+  margin-block-start: 1px;
+  margin-block-end: 1px;
+  padding-inline-start: 0;
+}
+
+.social-icon {
+  align-items: center;
+  justify-content: center;
+  height: 20px;
+  padding: 10px 0 0 20px;
+  transition: all 2s;
+}
+
+.social-icon:hover {
+  transform: rotate(360deg);
+}
+
+li.social-icon {
+  display: flex;
+}
+
+.projects {
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  padding: 16px;
+  width: 90%;
+  height: 600px;
+  left: calc(50% - 327px / 2);
+  top: 114px;
+  background: #fff;
+  border: 1px solid #dfe1e6;
+  box-sizing: border-box;
+  border-radius: 16px;
+  flex: none;
+  order: 0;
+  flex-grow: 0;
+  margin: 8%;
+}
+
+.title {
+  position: static;
+  width: 280px;
+  height: 25%;
+  left: 0;
+  top: 0;
+  font-family: 'Poppins', sans-serif;
+  font-style: normal;
+  font-weight: 700;
+  font-size: 32px;
+  line-height: 44px;
+  color: #172b4d;
+  padding-top: 10px;
+  padding-bottom: 20px;
+}
+
+.role {
+  display: flex;
+
+  /* grid-template-columns: 1fr 0.5fr 1.5fr 0.5fr 1.5fr 0fr; */
+}
+
```

Comment:
## [Optional changes]
- Kindly remove every commented code. This is to ensure and maintain a clean code base